### PR TITLE
PMM-8260 external access toggle

### PR DESCRIPTION
**Pull request by beatahandzelova**
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-dbaas-e2e-tests/detail/pmm2-dbaas-e2e-tests/169/tests

### CODE REVIEW

File: tests/DbaaS/pages/dbaasPage.js [hunk 1]
```diff
@@ -88,6 +88,9 @@ module.exports = {
           nodesFieldErrorMessage: '$nodes-field-error-message',
           nodesNumberField: '$nodes-number-input',
           resourcesPerNode: (clusterSize) => `//label[contains(text(), "${clusterSize}")]`,
+          dbClusterExternalAccessTooltip: locate('$expose-field-container').find('.css-sr6nr'),
```
Comment (puneet0191) [inline tests/DbaaS/pages/dbaasPage.js:91]: Please don't use these css locators, they are dynamic and always change

File: tests/DbaaS/pages/dbaasPage.js [hunk 2]
```diff
@@ -88,6 +88,9 @@ module.exports = {
...
+          dbClusterExternalAccessCheckbox: '$expose-switch',
+          dbClusterExternalAccessTooltipText: locate('div').withAttr({ class: 'popper' }),
```
Comment (puneet0191) [inline tests/DbaaS/pages/dbaasPage.js:93]: this class is correct?

**Review by puneet0191 — CHANGES_REQUESTED** [review]

File: tests/DbaaS/pages/dbaasPage.js (hunk 1 above)
Comment (beatahandzelova) [reply tests/DbaaS/pages/dbaasPage.js:91]: Ok, so what is the best way to grab this element please? I tried find it with 'svg' but it didn't work.

File: tests/DbaaS/pages/dbaasPage.js (hunk 2 above)
Comment (beatahandzelova) [reply tests/DbaaS/pages/dbaasPage.js:93]: I think it is, the test passed https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-dbaas-e2e-tests/detail/pmm2-dbaas-e2e-tests/129/pipeline.

**Review by puneet0191 — APPROVED** [review]

File: tests/DbaaS/pages/dbaasPage.js [hunk 3]
```diff
@@ -88,6 +88,9 @@ module.exports = {
           nodesFieldErrorMessage: '$nodes-field-error-message',
           nodesNumberField: '$nodes-number-input',
           resourcesPerNode: (clusterSize) => `//label[contains(text(), "${clusterSize}")]`,
+          dbClusterExternalAccessTooltip: `//div[@data-qa='expose-field-container']//div//div`,
```
Comment (yurkovychv) [inline tests/DbaaS/pages/dbaasPage.js:91]: could you please use `locate()` here?

File: tests/DbaaS/pages/dbaasPage.js [hunk 4]
```diff
@@ -88,6 +88,9 @@ module.exports = {
...
+          dbClusterExternalAccessCheckbox: '$expose-switch',
+          dbClusterExternalAccessTooltipText: locate('div').withAttr({ class: 'popper' }),
```
Comment (yurkovychv) [inline tests/DbaaS/pages/dbaasPage.js:93]: I think this can be changed to '`.popper'`

**Review by yurkovychv — APPROVED** [review]
Thanks

**Pull request merged**